examples/platform_channel/macos/Runner/MainFlutterWindow.swift

```diff
@@ -0,0 +1,84 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Cocoa
+import FlutterMacOS
+
+class MainFlutterWindow: NSWindow, FlutterStreamHandler, PowerSourceStateChangeDelegate {
+  private let powerSource = PowerSource()
+  private let stateChangeSource = PowerSourceStateChangeHandler()
```

Comment:
nit: `let stateChangeSource = PowerSourceStateChangeSource()` class naming

Reply:
Oh nice catch! I renamed the class partway through to avoid too many "source" names.